moviepy/audio/io/readers.py

```diff
@@ -239,7 +239,7 @@ def buffer_around(self, framenumber):
             current_f_end = self.buffer_startframe + self.buffersize
             if new_bufferstart < current_f_end < new_bufferstart + self.buffersize:
                 # We already have one bit of what must be read
-                conserved = current_f_end - new_bufferstart + 1
+                conserved = current_f_end - new_bufferstart
```

Comment:
The comment on line 241 seems to be talking about the `+ 1` that you've removed. Presumably the comment should be deleted as well if you've decided that it's unnecessary?

Reply:
"One bit" sounds ambiguous. I changed that to "part" to clarify.

Reply:
Yes, that makes more sense, thanks!